primitives/src/config.rs

```diff
@@ -3,60 +3,8 @@ use serde::{Deserialize, Serialize};
 use std::fs;
 use toml;
 
-pub const DEVELOPMENT_CONFIG: &str = r#"
-        max_channels = 512
-        channels_find_limit = 200
-        wait_time = 500
-
-        aggr_throttle = 0
-        events_find_limit = 100
-        msgs_find_limit = 10
-
-        heartbeat_time = 30000
-        health_threshold_promilles = 950
-        propagation_timeout = 1000
-
-        list_timeout = 5000
-        fetch_timeout = 5000
-        validator_tick_timeout = 5000
-
-        ip_rate_limit = { type = "ip", timeframe = 20000 }
-        SID_RATE_LIMIT = { type = 'sid', timeframe = 20000 }
-
-        ethereum_core_address = '0x333420fc6a897356e69b62417cd17ff012177d2b'
-        ethereum_network = 'goerli'
-    "#;
-
-pub const PRODUCTION_CONFIG: &str = r#"
-    # Maximum number of channels to return per request
-    max_channels = 512
-    
-    channels_find_limit = 512
-    wait_time = 500
-
-    aggr_throttle = 5000
-    events_find_limit = 100
-    msgs_find_limit = 10
-
-    heartbeat_time = 60000
-    health_threshold_promilles = 970
-    propagation_timeout = 3000
-
-    list_timeout = 10000
-    fetch_timeout = 10000
-    validator_tick_timeout = 10000
-
-    ip_rate_limit = { type = "ip", timeframe = 20000 }
-
-    ethereum_core_address = '0x333420fc6a897356e69b62417cd17ff012177d2b'
-    ethereum_network = 'homestead'
-    token_address_whitelist = ['0x89d24A6b4CcB1B6fAA2625fE562bDD9a23260359']
-
-    creators_whitelist = []
-    minimal_deposit = 0
-    minimal_fee = 0
-    validators_whitelist = []
-    "#;
+pub const DEVELOPMENT_CONFIG: &str = include_str!("../../docs/config/dev.toml");
```

Comment:
can't you parse the tomls directly and use the structs instead of `&str`s?

Reply:
it will need lazy_static I think

Reply:
For sure, already did that with `prep_db`

Reply:
I made the change to `lazy_static`

Reply:
now that I think about it, lazy_static is perhaps not the best idea, cause what happens if parsing fails? We blow up without an ability to handle as we want

Reply:
Yeah I want it to blow up, because that's the default configuration from Adex so if it's not correct then it should blow up

Reply:
that makes sense, cool; just make sure to use .expect rather than .unwrap